src/backend/distributed/sql/udfs/citus_finish_pg_upgrade/11.1-1.sql

```diff
@@ -104,7 +104,7 @@ BEGIN
     --
     -- register triggers
     --
-    FOR table_name IN SELECT logicalrelid FROM pg_catalog.pg_dist_partition
+    FOR table_name IN SELECT logicalrelid FROM pg_catalog.pg_dist_partition JOIN pg_class ON (logicalrelid = oid) WHERE relkind <> 'f'
```

Comment:
Now that we ship this change in 11.0-4 is this file really necessary?

I am unsure.